File: CS50X/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# get height of tower from user
while True:
      height = get_int("Height: ")
      if height > 0 and height < 9:
            break

#print height
for i in range(1, height + 1):
      print(" " * (height - i) + "#" * i)

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("how high is the tower?\n");
    }
    while (height < 1 || height >  8);

    for (int n = 0; n < height ; n++)
    {
        printf("\n");
    }
}
